.kitchen.yml

```diff
@@ -12,13 +12,21 @@ platforms:
   - name: debian-7.9
   - name: centos-6.7
   - name: centos-7.2
+  - name: windows-2012r2
+    driver:
+      box: mwrock/windows2012r2
```

Comment:
@anuriq I had to change this to mwrock/Windows2012R2 to get it to install. Can you update that? I'm guessing you didn't run into that because you already had the box installed.

Everything else looked good. I'll merge once you push the box name fix.